subworkflows/nf-core/fastq_removeadapters_merge/tests/main.nf.test

```diff
@@ -0,0 +1,205 @@
+nextflow_workflow {
+
```

Comment:
I think it might be nice to add a test skipping bbduk, trimgalore or trimmomatic (or all of them).  Right now, I believe the only step that is skipped/tested is `cutadapt` (in `sarscov2 - fastq - single-end - stub`) which isn't connected to any unique outputs that may disappear when it is skipped.